drivers/oled/oled_driver.c

```diff
@@ -154,6 +164,8 @@ bool oled_init(uint8_t rotation) {
     DISPLAY_OFFSET, 0x00,
     DISPLAY_START_LINE | 0x00,
     CHARGE_PUMP, 0x14,
+    // MEMORY_MODE is unsupported on SH1106 which only uses Page Addressing
+    // Mode, but sending it and the 0 byte following it seems to cause no harm.
```

Comment:
we could just define this out as the cmds array size is initialized by the parameters here...minor thing